saleor/graphql/schema.graphql

```diff
@@ -6518,8 +6518,54 @@ type ProductImage {
 }
 
 """Represents a product media."""
-type ProductMedia implements Node {
+type ProductMedia implements Node & ObjectWithMetadata {
   id: ID!
+
+  """List of private metadata items. Requires staff permissions to access."""
+  privateMetadata: [MetadataItem!]!
+
+  """
+  A single key from private metadata. Requires staff permissions to access.
+  
+  Tip: Use GraphQL aliases to fetch multiple keys.
+  
+  Added in Saleor 3.3.
+  
+  Note: this API is currently in Feature Preview and can be subject to changes at later point.
+  """
+  privateMetafield(key: String!): String
+
+  """
+  Private metadata. Requires staff permissions to access. Use `keys` to control which fields you want to include. The default is to include everything.
+  
+  Added in Saleor 3.3.
```

Comment:
For some reason you have incorrect label here. We need to add here a proper labels, also for some fields `added_in_3x` is missing (like `privateMetadata: [MetadataItem!]!`)

Reply:
There is an issue for that: https://github.com/saleor/saleor/issues/11702. WIll add something temporary to fix it. Thanks!